layouts/wrapper.html

```diff
@@ -33,12 +33,20 @@
 		{{/outputBlock}}
 	</head>
 	<body class="o-hoverable-on">
+
+		{{#if showSkipLinks}}
+			<a class="u-visually-hidden" href="#primary-nav">Skip to navigation</a>
+			<a class="u-visually-hidden" href="#content">Skip to content</a>
```

Comment:
It's not ideal, perhaps any app using next-express can inherit such a test somehow. 

I don't know if we have pre-launch checklists, it seems like the kind of thing we should have.